src/ddprof_worker.cc

```diff
@@ -702,15 +704,20 @@ DDRes ddprof_worker_process_event(const perf_event_header *hdr, int watcher_pos,
       if (wpid->pid) {
         uint64_t const mask = watcher->sample_type;
         perf_event_sample *sample = hdr2samp(hdr, mask);
+        LG_DBG("Processing event from watcher#%d@%ld", watcher_pos,
```

Comment:
I probably would not leave this logs (or create a new level of logging)

Reply:
Oooops, that should indeed not be there